[2024-08-11T10:31:25.121+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2024-08-10T00:00:00+00:00 [queued]>
[2024-08-11T10:31:25.154+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2024-08-10T00:00:00+00:00 [queued]>
[2024-08-11T10:31:25.155+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-08-11T10:31:25.161+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2024-08-11T10:31:25.162+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-08-11T10:31:25.201+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2024-08-10 00:00:00+00:00
[2024-08-11T10:31:25.214+0000] {standard_task_runner.py:55} INFO - Started process 86 to run task
[2024-08-11T10:31:25.219+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'scheduled__2024-08-10T00:00:00+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp8c_ij65m']
[2024-08-11T10:31:25.229+0000] {standard_task_runner.py:83} INFO - Job 11: Subtask create_table
[2024-08-11T10:31:25.455+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table scheduled__2024-08-10T00:00:00+00:00 [running]> on host ac5f4abca0b6
[2024-08-11T10:31:25.768+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2024-08-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-08-10T00:00:00+00:00
[2024-08-11T10:31:25.854+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2024-08-11T10:31:25.875+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            );
        , parameters: None
[2024-08-11T10:31:25.888+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2024-08-11T10:31:25.930+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20240810T000000, start_date=20240811T103125, end_date=20240811T103125
[2024-08-11T10:31:26.050+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2024-08-11T10:31:26.219+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
